code/game/objects/items/weapons/twohanded.dm

```diff
@@ -761,3 +763,66 @@
 				if(prob(4))
 					charged++
 					user.visible_message("<span class='notice'>The axe starts to emit an electric buzz!</span>")
+
+
+///Honkmother's Wrath///
+/obj/item/weapon/twohanded/chainsaw/honkmother
+	icon_state = "chainsaw0"
+	name = "Honkmother's Wrath"
+	desc = "The blade of the Honkmother herself. It hungers for blood, draining cowards who weild it, while rewarding thoes who kill in her name."
+	wieldsound = 'sound/weapons/chainsawstart.ogg'
+	armour_penetration = 60
+	attack_verb = list("SEEN", "LAUGHED", "PRANKED", "HONKED", "CUT ABOVE", "TIMBERED", "HATCHETED")
+
+/obj/item/weapon/twohanded/chainsaw/honkmother/update_icon()
+	if(wielded)
+		icon_state = "chainsaw[wielded]"
+	else
+		icon_state = "chainsaw0"
+
+/obj/item/weapon/twohanded/chainsaw/honkmother/attack(mob/target as mob, mob/living/user as mob)
+	if(wielded)
+		if (prob(50))
+			playsound(loc, 'sound/weapons/chainsaw.ogg', 100, 1, -1) //incredibly loud; you ain't goin' for stealth with this thing. Credit to Lonemonk of Freesound for this sound.
+		else
+			playsound(loc, 'sound/items/bikehorn.ogg', 100, 1, -1)
+		if(isrobot(target))
+			..()
+			return
+		if(!isliving(target))
+			return
+		else
+			target.Weaken(4)
+			..()
+			var/mob/living/carbon/human/H = user
+			if(target.client && (target.stat != DEAD))
+				user.reagents.add_reagent("adminordrazine", 0.4)
+				H.restore_blood()
+		return
+	else
+		playsound(loc, "swing_hit", 50, 1, -1)
+		return ..()
+
+/obj/item/weapon/twohanded/chainsaw/honkmother/wield(mob/living/carbon/user) //you can't disarm an active chainsaw, you crazy person.
+	..()
+	if (cursed == 0)
+		cursed = 1
+		playsound(get_turf(src), 'sound/effects/blobattack.ogg', 100, 1, -1)
+		user.visible_message("<span class='notice'>You feel your flesh twist and distort as the chainsaw replaces your hand attaching to your wrist!</span>")
+		user.mind.AddSpell(new /obj/effect/proc_holder/spell/bloodcrawl(null))
+		user.bloodcrawl = 1
+		processing_objects |= src
+		flags |= ABSTRACT
+
+/obj/item/weapon/twohanded/chainsaw/honkmother/process()
+	if(ishuman(loc))
+		var/mob/living/carbon/human/user = loc
+		if(user.species.exotic_blood)
+			user.vessel.remove_reagent(user.species.exotic_blood,2)
+		else
+			user.vessel.remove_reagent("blood",2)

```

Comment:
Hence why I put dealing oxy damage to IPCs, as far as I know they are the only exception. It will crit them at the same rate.